FacebookImagePicker/OLFacebookAlbumRequest.m

```diff
@@ -39,89 +35,79 @@ - (void)cancel {
 }
 
 - (void)getAlbums:(OLFacebookAlbumRequestHandler)handler {
-    __block BOOL runOnce = NO;
-    [UIApplication sharedApplication].networkActivityIndicatorVisible = YES;
-    [FBSession openActiveSessionWithReadPermissions:@[@"public_profile",  @"user_photos"]
-                                       allowLoginUI:YES
-                                  completionHandler:^(FBSession *session, FBSessionState state, NSError *error) {
```

Comment:
@dbotha Biggest change here is that we don't need to wrap everything inside a `FBSession` call.